packages/suite/src/hooks/wallet/useSendFormImport.ts

```diff
@@ -60,38 +80,28 @@ export const useSendFormImport = ({ network, tokens, localCurrencyOption, fiatRa
                     }
 
                     // calculate Fiat from Amount
-                    if (fiatRate?.rate) {
-                        const cryptoValue = shouldSendInSats
-                            ? formatAmount(output.amount, network.decimals)
-                            : output.amount;
-                        output.fiat =
-                            toFiatCurrency(
-                                cryptoValue,
-                                output.currency.value,
-                                fiatRate,
-                                2,
-                                false,
-                            ) || '';
+                    if (rate) {
+                        const formattedAmount = new BigNumber(
+                            shouldSendInSats
+                                ? formatAmount(output.amount, network.decimals)
+                                : output.amount,
+                        );
+                        const fiatValueBigNumber = formattedAmount.multipliedBy(rate);
+
+                        output.fiat = fiatValueBigNumber.toFixed(2) || '';
                     }
-                } else if (
-                    Object.keys(fiatCurrencies).find(c => c === currency) &&
-                    fiatRate?.rate
-                ) {
+                } else if (Object.keys(fiatCurrencies).find(c => c === currency) && rate) {
                     // csv amount in fiat currency
                     output.currency = { value: currency, label: currency.toUpperCase() };
                     output.fiat = item.amount || '';
                     // calculate Amount from Fiat
-                    const cryptoValue = fromFiatCurrency(
-                        output.fiat,
-                        currency,
-                        fiatRate,
-                        network.decimals,
-                        false,
-                    );
+                    const formattedLocalAmount = output.fiat.replace(',', '.');
+                    const amount = new BigNumber(formattedLocalAmount).div(rate);
+                    const amountStr = amount.toFixed(network.decimals);
```

Comment:
```suggestion
                    const amount = new BigNumber(formattedLocalAmount).div(rate).toFixed(network.decimals);
```
or better naming for both

Reply:
a1718ae7de5e8b020f8b1c9b763eee8039cc6bd1